### ISSUE TITLE
Immersive engineering Toolbox losing inventory 

### ISSUE THREAD

**Issue by aaronhowser1** [BluSunrize/ImmersiveEngineering#5230]
### Description of the issue:
FTBTeam/FTB-Modpack-Issues/issues/661
> Anything placed in toolbox vanishes from various changes to status. Not sure on exact triggers. Currently running on a dedicated server, have not tested in solo-world.

I can confirm it happens, though not 100% reproducable. I put some random blocks in it while in creative mode, switched to survival, and when I opened it the toolbox was empty.

### Versions & Modlist
ImmersiveEngineering-1.16.5-5.0.7-143
Forge 36.2.22
https://pastebin.com/khjmNTi3

**Comment by malte0811** [comment]
If it only happens when creative mode on dedicated servers is involved, see MinecraftForge/MinecraftForge#4580. We can essentially choose between crashing clients when people manage to put too much NBT in a toolbox, vanishing items in creative mode on dedicated servers, or a weird UUID-based system that I can guarantee will end up leaking memory. Of those, vanishing items in creative was the "least bad" option.

If you have a reproducible case that does not involve that, I'd be interested to hear it.

**Comment by BluSunrize** [comment]
Closing this due to lack of feedback.

**Comment by Shibva** [comment]
>... We can essentially choose between crashing clients when people manage to put too much NBT in a toolbox...

How is this a worse option compared to wiping their contents? Futhermore whats stopping you from making it block out other items that store items? The odds of someone triggering this only seem likely if they are using something like Create contraptions and storing multiple in there. Even then the chances of hitting that limit even wtih heavy NBT items are low and can be made to not be compatible with being stored in the toolbox

IMO, the NBT-crash option is annoying; yet expected. I mean, there's more slots in a shulker box that can cause that issue (yes the crates are arguable) but still it wont happen unless someone intentionally tries todo that. 

Its better if it did this like with any other mods storage. Hell, you could even add integration to XL packets that expands the packet limit up to 2GB for gods sake.

IN conclusion: it would be better if it just crashes/kicks players that put too many NBT things into it than to wipe. 
     IN addition to this, the toolbox and crates are not the only ones that do this, it does it to all things in IE that has some form of internal inventory such as MODIFICATIONS (yes even they wipe)
     
    Sorry if this sounds aggressive as I don't mean for it to come off as such; its more of frustration but even then it was not ment to be agressive.
